kratos/python_scripts/time_based_ascii_file_writer_utility.py

```diff
@@ -15,7 +15,8 @@ def __init__(self, model_part, params, file_header):
         default_settings = KratosMultiphysics.Parameters('''{
             "file_name"  : "",
             "output_path": "",
-            "write_buffer_size" : -1
+            "write_buffer_size" : -1,
+            "check_file_extension" : true
```

Comment:
```suggestion
            "file_extension" : "dat"
```

this way it would be easier for you to customize